locales/en-US/server.ftl

```diff
@@ -0,0 +1,214 @@
+// Global phrases shared across pages, prefixed with 'g'
+gMyShots = My Shots
+gHomeLink = Home
+gNoShots
+  .alt = No shots found
+gScreenshotsDescription = Screenshots made simple. Take, save, and share screenshots without leaving Firefox.
+
+// Creating page
+
+// Note: {$title} is a placeholder for the title of the web page
+// captured in the screenshot. The default, for pages without titles, is
+// creatingPageTitleDefault.
+creatingPageTitle = Creating {$title}
+creatingPageTitleDefault = page
+
+// Home page
+
+homePageDescription
+  .content = Intuitive screenshots baked right into the browser. Capture, save and share screenshots as you browse the Web using Firefox.
+homePageButtonMyShots = Go To My Shots
+homePageTeaser = Coming Soon…
+homePageDownloadFirefoxTitle = Firefox
+homePageDownloadFirefoxSubTitle = Free Download
+homePageGetStarted = Get Started
+// Note: do not translate 'Firefox Screenshots' when translating this string
+homePageHowScreenshotsWorks = How Firefox Screenshots Works
+homePageGetStarted = Get Started
+homePageGetStartedDescription = Find the new Screenshots icon on your toolbar. Select it, and the Screenshots menu will appear on top of your browser window.
```

Comment:
Add a comment explaining that Screenshots here is meant an abbreviation for "Firefox Screenshots", and should not be translated. BTW, the icon says Firefox Screenshots in my Nightly.

```
// Note: Screenshots is an abbreviation for Firefox Screenshots, and should not be translated.
```